### ISSUE TITLE
Support local installations of beautifiers

### ISSUE THREAD

**Issue by stevenzeck** [Unibeautify/unibeautify-cli#120]
_From @cdfa on December 11, 2018 13:49_

**Is your feature request related to a problem? Please describe.**
Yes, I was trying to get this working for a project where in which the dev-dependencies state everything I'm using to develop the project except for editor specific stuff, but unibeautify seems to be unable to find beautifiers (like beautifier-js-beautify) when they are installed locally.

**Describe the solution you'd like**
Support locally installed beautifiers.

**Describe alternatives you've considered**
- Having the beautifiers globally installed, but the other stuff locally, with a comment in the package.json saying the beautifiers need to be installed globally.

**Additional context**

_Copied from original issue: Unibeautify/unibeautify#213_

**Comment by stevenzeck** [comment]
OK, yes, the CLI only looks for globally installed beautifiers via [g-search](https://github.com/TremayneChrist/npm-g-search), then loads them via [requireg](https://github.com/h2non/requireg).  It would be an enhancement to add support for local installations.

**Comment by Glavin001** [comment]
Applicable code: 

https://github.com/Unibeautify/unibeautify-cli/blob/master/src/utils.ts#L3-L20

https://github.com/Unibeautify/unibeautify/blob/master/src/DependencyManager/NodeDependency.ts#L40-L52

It may require a change to Unibeautify core at https://github.com/Unibeautify/unibeautify/blob/master/src/DependencyManager/NodeDependency.ts#L44-L46 as well as CLI to search for global in addition to local.

Pull Requests very welcome!

**Comment by stevenzeck** [comment]
@Glavin001 is there a package out there that searches or lists locally installed packages?  I can't seem to find one.

**Comment by stevenzeck** [comment]
Not really.  Looking for something like this, only it searches in local:  https://github.com/TremayneChrist/npm-g-search.  Guess we could borrow that code and modify it to our needs...
